.editorconfig

```diff
@@ -0,0 +1,13 @@
+root = true
```

Comment:
thought: In the long term, we will want to enforce Swift Format.  How does the `.editorconfig` differ from the `.swiftformat` configuration file?  Which one takes precedence?

Would it make more sense to incorporate these setting in a single place, the `.swiftformat` file (or equivalent)?

Reply:
.editorconfig and .swift-format serve different purposes. One is _editor_ settings, the other is _formatter_ settings. The former applies to all text files under a directory hierarchy, not only to Swift. And the latter is much more comprehensive and concerned with the semantics and syntax of specific programming languages. They aren't in conflict; we should have both.

(Also, we shouldn't use .swiftformat; that's a third party tool. We should use .swift-format, which is the one that's officially part of the Swift ecosystem.)